gr-azure-software-radio/lib/difi_source_cpp_impl.cc

```diff
@@ -136,7 +235,7 @@ namespace gr {
         uint32_t idx = difi::DATA_START_IDX; //start after the data header
         while (items_written < out_items and idx < size_gotten)
         {
-          T test = d_unpacker(&d_buffer[idx]);
+          T test = d_unpacker(&d_packet_buffer[idx]);
```

Comment:
this was my bug, mind changing the T test to something more meaningful? like T sample?